apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -676,6 +708,35 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
         del data["submitted_answers"][answer_raw_name]
 
 
+def construct_feedback(
+    feedback_type: FeedbackType,
+    first_wrong: Optional[int],
+    group_belonging: dict[str, Optional[str]],
+    check_indentation: bool,
+    first_wrong_is_distractor: bool,
+) -> str:
+    if feedback_type is FeedbackType.NONE:
+        return ""
+
+    if first_wrong is None:
+        return FIRST_WRONG_FEEDBACK["incomplete"]
+    elif (
+        feedback_type is FeedbackType.FIRST_WRONG_VERBOSE and first_wrong_is_distractor
+    ):
+        return FIRST_WRONG_FEEDBACK["distractor-feedback"].format(str(first_wrong + 1))
+    else:
+        feedback = FIRST_WRONG_FEEDBACK["wrong-at-block"].format(str(first_wrong + 1))
+        has_block_groups = group_belonging != {} and set(group_belonging.values()) != {
```

Comment:
```suggestion
        has_block_groups = group_belonging and set(group_belonging.values()) != {
```
Nit: non-empty dicts evaluate to true.

Reply:
To someone who has written a lot of JavaScript, this looks so wrong (in JS, the empty object `{}` evaluates to true). Most of the other PL devs have done a lot of JS too so I'm not sold on this change. Do we have a convention for this in the codebase more generally?

Reply:
Yes; in Python code, we use idiomatic Python, which means using the form that Eliot is suggesting.